x/gov/types/errors.go

```diff
@@ -9,7 +9,7 @@ var (
 	ErrUnknownProposal       = sdkerrors.Register(ModuleName, 2, "unknown proposal")
 	ErrInactiveProposal      = sdkerrors.Register(ModuleName, 3, "inactive proposal")
 	ErrAlreadyActiveProposal = sdkerrors.Register(ModuleName, 4, "proposal already active")
-	// Errors 5 & 6 are reserved as legacy errors
+	// Errors 5, 6 & 7 are reserved as legacy errors
 	// See x/gov/types/v1beta1/errors.go
```

Comment:
Do we want to keep all errors in the same file or have it split?

Reply:
I think it makes sense to put them all together